samigo/samigo-app/src/webapp/WEB-INF/samigo.tld

```diff
@@ -140,6 +140,11 @@
     <required>true</required>
     <rtexprvalue>false</rtexprvalue>
   </attribute>
+  <attribute>
+    <name>timeUpMessage</name>
+    <required>true</required>
+    <rtexprvalue>false</rtexprvalue>
+  </attribute>
```

Comment:
does this really require a change to samigo.tld?!

Reply:
Yes because it is necessary specify the timeUpMessage attribute for samigo:timerBar